### PR TITLE
chore: New Kaizen team is now Codeowner of everything [KDS-147]

### PR DESCRIPTION
**Pull request by christian314159**
# Motivation

Our `CODEOWNERS` file specified the Frontend-Avocados as the main owner, a team that also includes the Kaizen advocates. As code owners get automatically tagged on PRs, some of the advocates subsequently (and rightfully) complained about getting too many notifications, e.g. about the many PRs opened by Renovate Bot.

But also the recent changes in our team structure made it necessary to revisit this file.

# Change

We now have a new [Github team](https://github.com/orgs/cultureamp/teams/kaizen-design-system/discussions) which mirrors the actual the Kaizen Design System team which is the sole owner of all code in this repository.

### CODE REVIEW

**Review by ActuallyACat — APPROVED** [review]

File: CODEOWNERS
Comment (HeartSquared) [inline CODEOWNERS:15]: We probably still want to allow the ops team to be able to approve CI changes for us right?

Comment (christian314159) [reply CODEOWNERS:15]: I am not sure this specific Github team even exists, I couldn't find it, so it's either gone or secret/invisible. But I also think we - the Kaizen team - own this repo, so any changes are our accountability & responsibility.

Nonetheless, asking for the Ops team's specific expertise in those areas is advised where needed, but we don't require a formal sign-off.

Comment (HeartSquared) [reply CODEOWNERS:15]: Not so much a formal sign-off, but it would be slightly annoying if I opened a PR related to CI changes and someone from Ops approved it but I can't merge because I need someone else from Kaizen to approve it too 😅 Unless that isn't what this does?

**Pull request merged**
